### ISSUE TITLE
Compliance/Penalty - Internal begin issuing Imposed penalty

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#81]
#### Description:

CAS staff need to be able to start the issuance process for an imposed penalty to issue one to an operator.

#### Acceptance Criteria:
As a CAS staff member, given that I need to start the imposed penalty issuance process, when I've clicked on the Issue New Penalty button on the Imposed Penalties grid, then I can begin filling in the operation identifying information for the penalty.

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=23878-8061&t=FlxrTh0WwwmbeGH9-1)

- [ ] 3 vertical tabs for issuing a penalty: Select Year & Operation, Enter Details and issue Penalty, Track Penalty Status & Payments

On the Select Year and Operation tab:

- [ ] Section title called Select Year & Operation
- [ ] Field label: Reporting Year/ Field format: Drop-down list for year
- [ ] Field label: Operator Name/ Field format: Drop-down list for operator
- [ ] Field label: Operation Name/ Field format: Drop-down list for operation (populated with operations associated with the selected operator)
- [ ] Back button to return to the Imposed Penalties grid
- [ ] Continue button that goes to the next tab, Enter Details and Issue Penalty

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #80 
Blocking #82 

#### Notes:

-
